features/steps/assign_incomplete_case.py

```diff
@@ -0,0 +1,76 @@
+from behave import *
+from selenium.webdriver.common.action_chains import ActionChains
+from create_eligible_finances import create_eligible_finance
+
+
+@given(u'case notes are empty')
+def step_impl(context):
+    notes = context.helperfunc.find_by_name('case.notes')
+    assert len(notes.text) == 0
+
+
+@step(u'I have created a user')
+def step_impl(context):
+    context.execute_steps(u'''
+        When I select 'Create new user'
+        And enter the client's personal details
+        And I click the save button on the screen
+    ''')
+
+
+@given(u'I have created a valid discrimination scope')
+def step_impl(context):
+    context.execute_steps(u'''
+        When I select ‘Create Scope Diagnosis'
+        And I select the categories Discrimination, Direct Discrimination, Disability, Work
+        Then I get an INSCOPE decision
+        And select 'Create financial assessment'
+    ''')
+
+
+@given(u'I am on the Diversity tab')
+def step_impl(context):
+    create_eligible_finance(context)
```

Comment:
Was just thinking since the `create_eligible_finance.py` basically is the same thing as the `means_test.py` would it be better to make use of mine since it's part of the tests?

Reply:
I feel that creating eligible finance would be used more than once, I would rather have a goto method than have to decipher the code across 11 different step definitions, one of which could be amended at any time.  Thanks for pointing to me what was in means test though.

Reply:
Yeah cool that makes sense. Thinking about it just now, I realised we only need one set of 'actual' tests. I thought having my additional assertions would make it a better reuse, but if my means test fails the whole tests fails so it doesn't matter.